<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LPConfig</class>
 <widget class="QDialog" name="LPConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Dataset</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Data Set:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_dataset">
       <property name="text">
        <string notr="true">sample</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_local">
      <attribute name="title">
       <string>Local Snapshots</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Schedule:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="combo_local_schedule">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Automatic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Daily</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hourly</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30 minutes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10 minutes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5 minutes</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="time_local_daily">
           <property name="maximumTime">
            <time>
             <hour>23</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
           <property name="minimumTime">
            <time>
             <hour>1</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
           <property name="displayFormat">
            <string>@ h AP</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_local_keep">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Keep:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QSpinBox" name="spin_local_numkeep">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>800</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_local_keepunits">
           <item>
            <property name="text">
             <string>Days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Total</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Excluded:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QListWidget" name="list_snap_excludes">
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QToolButton" name="tool_snap_addexclude">
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="icon">
              <iconset resource="lPreserve.qrc">
               <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
             </property>
             <property name="popupMode">
              <enum>QToolButton::InstantPopup</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tool_snap_remexclude">
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="icon">
              <iconset resource="lPreserve.qrc">
               <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
             </property>
             <property name="autoRaise">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_remote">
      <attribute name="title">
       <string>Replication</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Remote Servers</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QComboBox" name="combo_rep_hosts"/>
             </item>
             <item>
              <widget class="QToolButton" name="tool_rep_addhost">
               <property name="text">
                <string>SSH</string>
               </property>
               <property name="icon">
                <iconset resource="lPreserve.qrc">
                 <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tool_rep_addiscsi">
               <property name="text">
                <string>ISCSI</string>
               </property>
               <property name="icon">
                <iconset resource="lPreserve.qrc">
                 <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tool_rep_remhost">
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="lPreserve.qrc">
                 <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonIconOnly</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupReplicate">
             <property name="title">
              <string>Replication Settings</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>User Name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineUserName"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>SSH Port</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinPort">
                  <property name="maximum">
                   <number>999999</number>
                  </property>
                  <property name="value">
                   <number>22</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>Remote Dataset</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineRemoteDataset"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>Frequency</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item row="0" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QComboBox" name="combo_remote_schedule">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="sizeAdjustPolicy">
                      <enum>QComboBox::AdjustToContents</enum>
                     </property>
                     <item>
                      <property name="text">
                       <string>Sync with snapshots</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Daily</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Hourly</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>30 Minutes</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>10 Minutes</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Manual</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item>
                    <widget class="QTimeEdit" name="time_replicate">
                     <property name="displayFormat">
                      <string>@ h AP</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_6">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeType">
                      <enum>QSizePolicy::Preferred</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Excluded Data</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QListWidget" name="list_rep_excludes">
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QToolButton" name="tool_rep_addexclude">
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="lPreserve.qrc">
                 <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
               </property>
               <property name="popupMode">
                <enum>QToolButton::InstantPopup</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tool_rep_remexclude">
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="lPreserve.qrc">
                 <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
               </property>
               <property name="autoRaise">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_scrub">
      <attribute name="title">
       <string>Scrub</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupScrub">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Enable scheduled scrub</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QComboBox" name="combo_scrub_schedule">
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <item>
               <property name="text">
                <string>Daily</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Weekly</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Monthly</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Interval of Days</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Day of month</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Day of week</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string> Hour</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QSpinBox" name="spin_scrub_day_month">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>28</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="combo_scrub_day_week">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>Mondays</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tuesdays</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Wednesdays</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Thursdays</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Fridays</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Saturdays</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Sundays</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="time_scrub">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumTime">
               <time>
                <hour>23</hour>
                <minute>0</minute>
                <second>0</second>
               </time>
              </property>
              <property name="minimumTime">
               <time>
                <hour>1</hour>
                <minute>0</minute>
                <second>0</second>
               </time>
              </property>
              <property name="displayFormat">
               <string>@ h AP</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="tool_apply">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="icon">
        <iconset resource="lPreserve.qrc">
         <normaloff>:/images/backup-ok.png</normaloff>:/images/backup-ok.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tool_cancel">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="icon">
        <iconset resource="lPreserve.qrc">
         <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lPreserve.qrc"/>
 </resources>
 <connections/>
</ui>
